ci/build_wheel.sh

```diff
@@ -6,6 +6,10 @@ set -euo pipefail
 package_name=$1
 package_dir=$2
 
+# Clear out system ucx files to ensure that we're getting ucx from the wheel.
+rm -rf /usr/lib64/ucx
+rm -rf /usr/lib64/libuc*
+
```

Comment:
I added this because [UCX is installed in the wheel containers](https://github.com/rapidsai/ci-imgs/blob/main/ci-wheel.Dockerfile#L140) and I wanted to make sure that we weren't accidentally depending on it. I would prefer to leave this here until we update the CI containers to be safe, but we need to update other RAPIDS repos to use the new UCX wheels before we can update the containers.